ui/app/models/config-ui/message.js

```diff
@@ -11,10 +11,22 @@ import { withFormFields } from 'vault/decorators/model-form-fields';
 const validations = {
   title: [{ type: 'presence', message: 'Title is required.' }],
   message: [{ type: 'presence', message: 'Message is required.' }],
+  link: [
+    {
+      validator(model) {
+        if (!model?.link) return true;
+        const [linkTitle] = Object.keys(model.link);
+        const [linkHref] = Object.values(model.link);
```

Comment:
Also what happens if a user has a URL but doesn't want a title? Should we validate for both?

Reply:
`const [title, href] = Object.entries(model.link)[0]` doesn't work when the object is `{"": ""}` since `Object.entries(model.link)` is an empty array. I did however add this suggested change 
```
// if there is a title or href, validate both have values otherwise return true return (title || href) ? title && href : true
```